netbout-web/src/main/java/com/netbout/rest/bout/TkAttach.java

```diff
@@ -133,7 +122,27 @@ public Response act(final Request req) throws IOException {
             throw new RsFailure(ex);
         }
         FileUtils.forceDelete(temp);
-        throw new RsForward(new RsFlash(msg.toString()));
+        bout.messages().post(msg.toString());
+        return new RsForward(new RsFlash(msg.toString()));
```

Comment:
@pinaf In that case, we have to make the assertion in the try..catch block in the test and expect `RsForward` to be thrown. This was your [remark](https://github.com/yegor256/netbout/pull/837#discussion_r44881169) earlier.

Actually, I would say that this is more correct approach than throwing `RsForward` (an exception). `TkAttach` is used only in one place, in [`TkBout`](https://github.com/yegor256/netbout/blob/master/netbout-web/src/main/java/com/netbout/rest/bout/TkBout.java), and behavior-wise there is no difference there between returning or throwing the `RsForward`.

Please feel free to correct me again if my understanding is incorrect.